test/fixtures/linter/autofix/ansi/007_with_clause/after.sql

```diff
@@ -1,5 +1,5 @@
 -- Dealing with complicated indents before with clauses.
     	WITH cte as (
             select a from tbla
-        )
+)
```

Comment:
I think this test case is a bit pointless. It's only one rule. It shouldn't really be in this folder and should be in the rule YAML file.

However, now we are making L003 do the indentation I think we could leave it here, if we added L003 to the test config and then make sure that the closing bracket is aligned. In which case we can revert this change.

Reply:
Added the L003 and changed the `after.yml`